packages/web/app/src/lib/kit/types/json.ts

```diff
@@ -0,0 +1,19 @@
+export namespace JSON {
+  export const encode = <value extends Value>(value: value): string => {
+    return globalThis.JSON.stringify(value);
+  };
+
+  export const encodePretty = <value extends Value>(value: value): string => {
+    return globalThis.JSON.stringify(value, null, 2);
+  };
+
+  export const decode = (value: string): Value => {
+    return globalThis.JSON.parse(value);
```

Comment:
Same with decoding, if we expect deserialized json to have some shape, we can just slap Zod on it and use JSON.parse

Reply:
`JSON.parse` results in an `any` return type. This is generally even worse than the problems with `any` on the input that I mentioned in https://github.com/graphql-hive/console/pull/6378#discussion_r1920311753, because that `any` propagates into all downstream code.

I do not think `any` should be a casually accepted type in our code base. I think its removal will result in code that is easier for us to read and write, fewer bugs for end-users, and lowered time to delivery of features.

Reply:
We could turn `JSON.parse` type definition to `unknown` globally and go from there.

Reply:
even more type safe

Reply:
It is not more type safe, it just more annoying. The result of `JSON.parse` has a known type, which I have defined in this PR.